### DISCUSSION TITLE
how to unite/merge all paths in a glyph into as few paths as possible.

### DISCUSSION THREAD

**Discussion by rickidevsos** [fonttools/fonttools#3552] [Q&A] [answered]
I have written a script that applies an 'X' over my fonts, I am trying to merge the 'X' with each glyph so that it cannot be separated. I am only able to achieve this so far using PyClipper which destroys the character by transforming it into a polygon. 

Is there a way in fonttools (with/without extensions) that I can merge paths in a glyph so they become inseparable?

**Comment by behdad** [comment, marked as answer]
Check out `fontTools.ttLib.removeOverlaps`.

**Reply by rickidevsos** [reply]
Thanks @behdad you made my weekend! This worked
